public/lib/content-service.js

```diff
@@ -16,11 +16,17 @@ angular.module('wfContentService', ['wfHttpSessionService', 'wfVisibilityService
             class ContentService {
 
                 provideFormats(featureSwitches){
-                    const articleFormats = {
+                    let articleFormats = {
```

Comment:
`const` only prevents reassignment using the assignment operator (i.e. =) so this isn't actually necessary!

Reply:
Huh, that's right! Somehow haven't come across this particular operation before, but good to know 👍